lib/wallet/src/Cardano/Wallet/Write/Tx.hs

```diff
@@ -454,6 +525,64 @@ downcastTxOut (TxOutInRecentEra addr val Alonzo.NoDatum Nothing)
 downcastTxOut (TxOutInRecentEra addr val (Alonzo.DatumHash dh) Nothing)
     = Right $ Alonzo.TxOut addr val (SJust dh)
 
+--
+-- MinimumUTxO
+--
+
+-- |
+--
+-- prop> forall out.
+-- prop> let
+-- prop>     c = computeMinimumCoinForUTxO out
+-- prop> in
+-- prop>     forall c' > c. satisfiesMinimumUTxO (out { coin = c' })
```

Comment:
Need to tweak the formatting a bit.

And the property check coverage is actually lacking. I thought we'd get it for free from prop_balanceTransactionValid but this is not the case currently at least.

Regardless, `computeMinimumCoinForTxOut` should definitely be property tested on its own, and possibly `increaseZeroAdaOutputs` too.

Reply:
Tweaked doc comment & added property for `computeMinimumCoinForTxOut`.